tests/integration/utils.py

```diff
@@ -242,26 +242,34 @@ def assert_content_manifest_schema(response_data):
     ), f"ICM data not valid for schema at {response_data['metadata']['icm_spec']}: {response_data}"
 
 
-def assert_element_from_response(response_data, expected_element, element_name):
+def assert_elements_from_response(response_data, expected_response_data):
     """
-    Check element from the response data.
+    Check elements from the response data.
 
     In case the expected element is a list, every element in the list will be checked
     (otherwise only equality between expected_element and element from response).
+    Elements "packages" and "dependencies" will be sorted for
+    response_data and expected_response_data.
 
     :param dict response_data: response data from the Cachito request
-    :param expected_element: expected content of particular element in response
-    :param str element_name: name of the element (ex. "dependencies", "packages")
+    :param expected_response_data: expected content of particular elements in response:
+        {<element_name> : <expected_data>}
     """
-    response_element = response_data[element_name]
-    if isinstance(response_element, list):
-        assert len(response_element) == len(expected_element)
-        for e in expected_element:
-            assert (
-                e in response_element
-            ), f"#{response_data['id']}: {e} should be in {response_element}"
-    else:
-        assert response_element == expected_element
+    sort_pkgs_and_deps_in_place(response_data["packages"], response_data["dependencies"])
+    if "packages" in response_data and "dependencies" in expected_response_data:
+        sort_pkgs_and_deps_in_place(
+            expected_response_data["packages"], expected_response_data["dependencies"]
+        )
+    for element_name, expected_data in expected_response_data.items():
+        response_element = response_data[element_name]
+        if isinstance(response_element, list):
+            assert len(response_element) == len(expected_data)
+            for e in expected_data:
+                assert (
+                    e in response_element
+                ), f"#{response_data['id']}: {e} should be in {response_element}"
+        else:
+            assert response_element == expected_data
```

Comment:
If packages and deps are already sorted, then we don't need special handling for lists, right?

Reply:
Done, thank you!